src/main/java/uk/gov/hmcts/reform/idam/web/helper/AuthHelper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.idam.web.helper;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class AuthHelper {
+
+    private final boolean useSecureCookie;
+
+    public AuthHelper(@Value("${authentication.secureCookie}") boolean useSecureCookie) {
+        this.useSecureCookie = useSecureCookie;
+    }
+
+    public List<String> makeCookiesSecure(List<String> cookies) {
```

Comment:
Suggestion: the method name suggests that it makes cookies secure, but it also sets the path and adds `HttpOnly` regardless of the flag value. Is there maybe a better name for it?

Reply:
@radoslaw-orlowski-amido all those flags make the cookie secure to prevent cross site attacks and the cookie being transmitted over an unsecure connection.